integrations/jenkins/documentation.yaml

```diff
@@ -0,0 +1,77 @@
+exporter_type: included
+app_name_short: Jenkins
+app_name: {{app_name_short}}
+app_site_name: Jenkins
+app_site_url: https://www.jenkins.io/
+exporter_name: the Jenkins exporter
+exporter_pkg_name: jenkins
+exporter_repo_url: https://github.com/jenkinsci/prometheus-plugin
+additional_prereq_info: |
+  {{exporter_name}} exposes Prometheus-format metrics when the [metrics](https://plugins.jenkins.io/metrics/)
+  and [prometheus](https://plugins.jenkins.io/prometheus/) plugins are installed.
+  You can run the following checks to verify that {{app_name_short}} is exposing Prometheus-format metrics.
+
+   *  To determine if {{exporter_name}} is exposing metrics, run the following command:
+
+      <pre class="devsite-click-to-copy">
+      kubectl -n <namespace_name> port-forward <jenkins_pod_name> <jenkins_port>
+      curl localhost:<jenkins_port>/prometheus
+      </pre>
```

Comment:
OK, like the Kibana case, or our existing docs for [zookeeper](https://cloud.google.com/stackdriver/docs/managed-prometheus/exporters/zookeeper) / [consul](https://cloud.google.com/stackdriver/docs/managed-prometheus/exporters/consul), beyond saying "Jenkins exports Prometheus-format metrics when ... are installed, here's how you verify this", let's give a couple more hints of how to install these plugins in the first place.

Even though https://plugins.jenkins.io/prometheus has a "How to install" link, the 3 options there don't seem 100% applicable for jenkins running on a GKE cluster... I got confused myself when I tried to set this up and test these instructions, and came up with the following draft -- are there other methods you would recommend? 

---

If you installed [the Jenkins community helm chart](https://github.com/jenkinsci/helm-charts/tree/main/charts/jenkins), you may include the Prometheus plugin in the `controller.installPlugins` field. Alternatively, you may create your own {{app_name_short}} container image with the following Dockerfile:

```
FROM jenkins/jenkins:lts
RUN jenkins-plugin-cli --plugins prometheus metrics
```

Reply:
Also if we are specifying 2.0.11 above, should we specify `prometheus:2.0.11` etc in the sample Dockerfile?